src/app/api/create-meme/route.ts

```diff
@@ -134,9 +128,38 @@ export async function GET(request: Request) {
     // Create form data
     const formData = new FormData();
 
-    // Fetch image from URL and create file
-    const imageResponse = await fetch(imageUrl);
-    const imageBlob = await imageResponse.blob();
+    const convertImageUrlToBase64 = async (
+      imageUrl: string
+    ): Promise<string> => {
+      try {
+        const response = await fetch(imageUrl);
+        if (!response.ok) {
+          throw new Error(`Failed to fetch image: ${response.statusText}`);
+        }
+        const arrayBuffer = await response.arrayBuffer();
+        const base64String = Buffer.from(arrayBuffer).toString("base64");
+        const mimeType = response.headers.get("content-type") || "image/jpeg"; // Default to JPEG if MIME type is not available
+        return `data:${mimeType};base64,${base64String}`;
+      } catch (error) {
+        console.error("Error converting image to base64:", error);
+        throw error;
+      }
+    };
+
+    const dataUriToBlob = (dataUri: string): Blob => {
```

Comment:
These static functions should not be declared in the GET request.  Same for all of the constant values.